python/lib/dependabot/python/version.rb

```diff
@@ -29,7 +29,7 @@ def self.correct?(version)
 
       def initialize(version)
         @version_string = version.to_s
-        version, @local_version = version.split("+")
```

Comment:
We should add a test for this in a follow up PR 🙏

Reply:
@yeikel if you're up for it, I'd be more than happy to review and merge it!